File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/CuentasPropiasDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface CuentasPropiasDAO
    {
        int Insertar(CuentasPropiasDTO cuentaPropia);
        int Modificar(CuentasPropiasDTO cuentaPropia);
        int eliminar(CuentasPropiasDTO cuentaPropia);
        int eliminarLogico(CuentasPropiasDTO cuentaPropia);
        CuentasPropiasDTO ObtenerPorId(int cuentaPropiaId);
        IList<CuentasPropiasDTO> ListarTodos();

    }
}

File: BackEnd/Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/CuentasPropiasDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.ArrayList;
import java.util.List;

import pe.edu.pucp.softpac.model.CuentasPropiasDTO;
import pe.edu.pucp.softpac.model.UsuariosDTO;

public interface CuentasPropiasDAO {
    public Integer insertar(CuentasPropiasDTO cuentaPropia);
    
    public CuentasPropiasDTO obtenerPorId(Integer cuentaPropiaId);
    
    public List<CuentasPropiasDTO> listarTodos();
    
    public Integer modificar(CuentasPropiasDTO cuentaPropia);
    
    public Integer eliminar(CuentasPropiasDTO cuentaPropia);

    public Integer eliminarLogico(CuentasPropiasDTO cuentaPropia);
}
